chia/full_node/full_node_store.py

```diff
@@ -45,8 +45,11 @@ class FullNodeStore:
     candidate_blocks: Dict[bytes32, Tuple[uint32, UnfinishedBlock]]
     candidate_backup_blocks: Dict[bytes32, Tuple[uint32, UnfinishedBlock]]
 
-    # Header hashes of unfinished blocks that we have seen recently
-    seen_unfinished_blocks: Set[bytes32]
+    # Block hashes of unfinished blocks that we have seen recently. This is
+    # effectively a Set[bytes32] but in order to evict the oldest items first,
+    # we use a Dict that preserves insertion order, and remove from the
+    # beginning
+    seen_unfinished_blocks: Dict[bytes32, bool]
```

Comment:
Does this work?  Seems _slightly_ more explicit that the value is irrelevant.

```suggestion
    seen_unfinished_blocks: Dict[bytes32, None]
```

Reply:
I got errors when using `None`. I think it has to do with being able to distinguish between a key existing or not, in the return value of `.get()`. But the error said something about `Sized`

Reply:
mypy seems ok with it at this point and I don't see code using the value at all, as you would expect.

Reply:
I must have been mistaking this for some unrelated error. You're right, it seems to work